pkg/api/slang.go

```diff
@@ -11,6 +11,62 @@ import (
 	"github.com/google/uuid"
 )
 
+// todo should be SlangFileDef method
+func BuildOperator(sf *core.SlangFileDef) (*core.Operator, error) {
+	if !sf.Valid() {
+		if err := sf.Validate(); err != nil {
+			return nil, err
+		}
+	}
+
+	blueprints := []core.Blueprint{}
+	blueprints = append(blueprints, sf.Blueprints.Elementary...)
```

Comment:
@tidify the system bundles do no longer have that structure, also @jm9e this export format from the studio.

Reply:
@kairichard fixed it